azerothcore/azerothcore-wotlk
# fix(DB/Item) Flamewrought Key zhCN translation

## Changes Proposed:
- `Flamewrought Key` gets a zhCN translation


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4364


## Tests Performed:
- none


## How to Test the Changes:
- Apply the SQL changes and run the core with a zhCN client. 
- .additem 30622 for Alliance
- .additem 30637 for Horde
- make sure the items have the proper translation


## Known Issues and TODO List:

- [x] Can someone confirm this is actually the right translation?


## Target Branch(es):


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
